ui-v2/app/templates/components/aria-menu.hbs

```diff
@@ -0,0 +1 @@
+{{yield (action 'change') (action 'keypress') (concat 'component-aria-menu-trigger-' guid) (concat 'component-aria-menu-menu-' guid) (if expanded 'true' undefined)}}
```

Comment:
Yeah agree somewhat here, its a fine line between giving the user control over the name of the actions/variables vs lots of arguments getting out of hand and relying on order as I do here. I'm not 100% sure what to call these myself yet, so it would be good to let it sit a little while. I can revisit this before GA though, or when I next make another aria component which I guess will follow similar patterns.